pandas/core/arrays/string_arrow.py

```diff
@@ -627,28 +627,25 @@ def _str_map(
                 na_value = np.nan
             else:
                 na_value = False
-            try:
```

Comment:
@phofl 

If you're free, could you review the changes to the Arrow strings here?

I think what I have here is correct, but not too familiar with the arrow strings.

Reply:
@mroeschke 

Are you able to review this bit instead?

I'd like to get this in soon.